### ISSUE TITLE
Pulid not working - xFormers issue

### ISSUE THREAD

**Issue by bipinpeter** [ToTheBeginning/PuLID#153]
Here is the error message I get

ApplyPulid
No operator found for `memory_efficient_attention_forward` with inputs:
query : shape=(1, 577, 16, 64) (torch.float16)
key : shape=(1, 577, 16, 64) (torch.float16)
value : shape=(1, 577, 16, 64) (torch.float16)
attn_bias : <class 'NoneType'>
p : 0.0
`decoderF` is not supported because:
xFormers wasn't build with CUDA support
attn_bias type is <class 'NoneType'>
operator wasn't built - see `python -m xformers.info` for more info
`flshattF@0.0.0` is not supported because:
xFormers wasn't build with CUDA support
operator wasn't built - see `python -m xformers.info` for more info
`cutlassF` is not supported because:
xFormers wasn't build with CUDA support
operator wasn't built - see `python -m xformers.info` for more info
`smallkF` is not supported because:
max(query.shape[-1] != value.shape[-1]) > 32
xFormers wasn't build with CUDA support
dtype=torch.float16 (supported: {torch.float32})
has custom scale
operator wasn't built - see `python -m xformers.info` for more info
unsupported embed per head: 64

**Comment by timsplinge** [comment]
TL;DR update pip in the python_embedded folder. Then try installing xformers with --no-deps


I don't know what I'm doing (clearly) but I did three things, one or more of which cured this problem for me:-

uninstalled and reinstalled xformers, this time with --nodeps
( pip install xformers --no-deps )

uninstalled xformers from the python_embedded folder

then updated pip from the python_embedded folder

then reinstalled xformers from the python_embedded folder with --no-deps
( python.exe -m pip install xformers --no-deps )
